FWCore/PyDevParameterSet/BuildFile.xml

```diff
@@ -2,7 +2,7 @@
 <use   name="FWCore/ParameterSet"/>
 <use   name="FWCore/Utilities"/>
 <use   name="py2-pybind11"/>
-<use   name="python"/>
+<use   name="python3"/>
```

Comment:
ok, so whatever uses PyDevParameterSet from now onwards is effectively bound to python3